shared/fs/common/pie-slice.tsx

```diff
@@ -73,10 +73,8 @@ const styles = Styles.styleSheetCreate({
   },
   leftUnfilled: {
     ...stylePieHalf,
-    // -1 is a workaround for a rendering issue where the filled part of the
```

Comment:
I still prefer the slightly rounded look to the bleeding-through look, especially since how bad the bleeding is depends on your resolution. In your screenshot it's not too bad:
<img width="67" alt="image" src="https://user-images.githubusercontent.com/4698687/59929370-b0500d00-940e-11e9-947b-87e5f4ec734c.png">

but still I don't like it. I'd rather keep the rounded rectangles and hope that they're not noticeable when the background is correct.

Reply:
The rectangles look pretty awful. Much more noticeable than the slight spillage. But I will investigate another approach.

Reply:
I think the rectangles only look awful when there is a lot of contrast between the background color and the main circle color. If we can't avoid that, though, I agree that the spillage is preferable and probably ok.